<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tj2" >
        <node name="tj2_gameobjects" pkg="tj2_gameobjects" type="tj2_gameobjects_node.py" output="screen">
            <!-- <rosparam file="$(find tj2_gameobjects)/config/cargo_2022.yaml" command="load"/> -->
            <rosparam file="$(find tj2_gameobjects)/config/powercell_2021.yaml" command="load"/>
            <remap from="pf_particles" to="gameobjects/pf_particles"/>
            <param name="velocity_smooth_k" value="0.0"/>
            <remap from="odom" to="/tj2/odom"/>
        </node>
    </group>
</launch>
